p2p/pex/addrbook.go

```diff
@@ -698,7 +698,7 @@ func (a *addrBook) randomPickAddresses(bucketType byte, num int) []*p2p.NetAddre
 	case bucketTypeOld:
 		buckets = a.bucketsOld
 	default:
-		panic("unexpected bucketType")
+		panic(errUnexpectedBucketType)
```

Comment:
Not sure what this changes since it is a panic.

Reply:
Reverted.